maps/LakeOfRage.asm

```diff
@@ -180,9 +180,8 @@ Randomizer_RedGyaradosSpecies::
 .NotBeaten:
 	reloadmapafterbattle
 	opentext
-	giveitem RED_SCALE
+	verbosegiveitem RED_SCALE
 	waitsfx
-	writetext LakeOfRageGotRedScaleText
```

Comment:
For now I suggest keeping this in case its decided that this change should be locked behind a flag. I suspect that it probably won't need to be though, so once this gets properly merged into the main branch for speedchoice V7 we should propose to delete this.